dev-games/ogre/metadata.xml

```diff
@@ -77,18 +77,54 @@ Exporters
     * Maya (meshes)
   </longdescription>
   <use>
-    <flag name="boost">enable boost support</flag>
+    <flag name="boost" restrict="&lt;dev-games/ogre-1.10">
```

Comment:
You don't have to restrict flag name descriptions just because they are only active for some versions. Please make use of this only if the same flag has conflicting meaning over different versions.

Reply:
Oh dammit!!! What on earth were I thinking? I guess I got into some sort of "stride" or something like that.
*gnarf*...
...fixed!